src/Microsoft.TestPlatform.ObjectModel/Adapter/Interfaces/ITestExecutor2.cs

```diff
@@ -0,0 +1,34 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+namespace Microsoft.VisualStudio.TestPlatform.ObjectModel.Adapter
+{
+    using System;
+    using System.Collections.Generic;
+
+    /// <summary>
+    /// Defines the test executor which provides capability to run tests.  
+    /// 
+    /// A class that implements this interface will be available for use if its containing 
+    //  assembly is either placed in the Extensions folder or is marked as a 'UnitTestExtension' type 
+    //  in the vsix package.
+    /// </summary>
+    public interface ITestExecutor2 : ITestExecutor
+    {
+        /// <summary>
+        /// Asks the adapter about attaching to the default test host.
```

Comment:
Please change this to: `Indicates whether or not the default test host process should be attached to.`

Reply:
Done.